### PR TITLE
CHANGELOG and version changes for release 2.20

### PR DESCRIPTION
**Pull request by satadruroy**
Preparing for 2.20 release.

### CODE REVIEW

**Review by gaktive — APPROVED** [review]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## [2.20.0] - 2020-01-10
+
+## Added
+- Added/fixed podAntiAffinity rules for various roles
+- Set topologyKey for podAntiAffinity rule to kubernetes.io/hostname and made it configurable
```
Comment (jandubois) [inline CHANGELOG.md:7]: I wouldn't mention that it is configurable; we don't document how to configure it, and don't expect people to modify it, except (maybe) under guidance from our service/support people.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+- Advise operator of the need for CSR approval for Eirini internal registry in helm NOTEs 
+
+
+## Changed
+- Bumped autoscaler to hotfix release 1.2.4
+- Bumped cf-deployment to 12.17
+- Bumped cflinuxfs3 release to 0.153.0
+- Bumped eirini release to 0.0.26
+- Bumped go-buildpack release to 1.9.4
+- Bumped nginx-buildpack release to 1.1.1
```
Comment (jandubois) [inline CHANGELOG.md:17]: 1.1.2

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+- Bumped java-buildpack release to 4.27.0
+- Bumped nodejs-buildpack release to 1.7.7
+- Bumped php-buildpack release to 4.4.2
+- Bumped python-buildpack release to 1.7.3
+- Bumped staticfile-buildpack release to 1.5.2 
+- Bumped ruby-buildpack release to 1.8.3
+- Bumped scf-helper-release to 1.0.9
```
Comment (jandubois) [inline CHANGELOG.md:24]: 1.0.13

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+- Bumped SLE12 and SLE15 stack
+- Optimized startup time for various roles
+ 
+## Fixed
+- Avoid pod restarts for many roles during UAA and SCF startup
+- Expand GoRouter ciphersuite to be compatible with a wider range of clients
+- Use apps/v1 apiVersion for Statefulsets
```
Comment (jandubois) [inline CHANGELOG.md:31]: Better: "Make helm chart compatible with kubernetes 1.16"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+- Use system cert store if UAA_CA_CERT is empty
```
Comment (jandubois) [inline CHANGELOG.md:32]: s/empty/not specified/

**Review by jandubois — CHANGES_REQUESTED** [review]

**Pull request merged**